src/elfpy/pricing_models.py

```diff
@@ -69,7 +69,7 @@ def norm_days(days, normalizing_constant=365):
         return days / normalizing_constant
 
     @staticmethod
-    def _stretch_time(time, time_stretch=1):
+    def stretch_time(time, time_stretch=1):
```

Comment:
Again, this shouldn't block a merge, but don't we want to remove this from the class and put it in a standalone module?

Reply:
Moved these functions to the time utils file